### PR TITLE
Refactored and completed tests.

### PR DESCRIPTION
**Pull request by carloskelly13**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,9 +5,9 @@
   "repository": "https://github.com/ModusCreateOrg/ionic-seed",
   "license": "MIT",
   "dependencies": {
-    "angular": "^1.4.8",
-    "angular-animate": "^1.4.8",
-    "angular-sanitize": "^1.4.8",
+    "angular": "^1.4.3",
```
Comment (dmackerman) [inline package.json:8]: Why'd we go down to 1.4.3? Just curious.

File: src/app/app.directives.js [hunk 2]
```diff
@@ -1,5 +1,7 @@
 import userMenu from './components/directives/userMenu/userMenu';
 
 export default angular
-    .module('ionicSeed.directives', [])
+    .module('ionicSeed.directives', [
+        'ionic'
```
Comment (dmackerman) [inline src/app/app.directives.js]: Necessary? Shouldn't it already be imported at the `ionicSeed` level?

File: package.json (hunk 1 above)
Comment (carloskelly13) [inline package.json:8]: That matches the version bundled with Ionic.

File: src/app/app.spec.js [hunk 3]
```diff
@@ -1,37 +1,47 @@
...
 // injected by Karma. no need to import them manually for each test.
-import AppController from '../../src/app/app.controller';
-import AppConfig from '../../src/app/app.config';
+import App from './app';
 
 describe('Application Tests', () => {
+    let sandbox, stubs, controller;
+
+    before(() => {
+        sandbox = sinon.sandbox.create();
```
Comment (dmackerman) [inline src/app/app.spec.js:9]: I like this a lot, didn't know it existed!

File: package.json (hunk 1 above)
Comment (dmackerman) [inline package.json:8]: :+1:

**Pull request merged**